apps/frontline-widgets/src/app/messenger/hooks/useConnect.tsx

```
@@ -60,35 +67,69 @@
+      // Get fresh values from messengerData and localStorage
+      const currentCachedCustomerId = getLocalStorageItem('customerId');
+      const email = messengerData?.email;
+      const phone = messengerData?.phone;
+      const code = messengerData?.code;
+      const customData = messengerData?.data;
+      const companyData = messengerData?.companyData;
 
       let visitorId;
 
       const { getVisitorId } = await import('@libs/utils');
       visitorId = await getVisitorId();
```

ESLint

[error] 'visitorId' is never reassigned. Use 'const' instead.

(prefer-const)